atlasdb-commons/src/main/java/com/palantir/common/compression/EnumClientCompressor.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.common.compression;
+
+import java.util.zip.GZIPInputStream;
+
+import net.jpountz.lz4.LZ4BlockInputStream;
+
+public enum EnumClientCompressor {
```

Comment:
you don't need to preface all enum types with `Enum` - recommend just calling this type `ClientCompressor`